libs/server/Storage/SizeTracker/CacheSizeTracker.cs

```diff
@@ -75,10 +76,10 @@ public CacheSizeTracker(TsavoriteKV<byte[], IGarnetObject> store, LogSettings lo
             }
         }
 
-        public void Start()
+        public void Start(CancellationToken token)
         {
-            mainLogTracker.Start();
-            readCacheTracker?.Start();
+            mainLogTracker.Start(token);
```

Comment:
why is there a main log tracker always? arent' there cases where the main log is fixed in size for the server in which case we can avoid having the tracking running.

Reply:
The containing class CacheSizeTracker is instantiated only when Object Store total memory size config is set. Here, the mainLogTracker variable is to track the size of the object store size whereas the readCacheTracker is for its corresponding read cache that is optional based on if read cache has been enabled. So, whenever object store managed memory size is to be tracked that is done using the tracker called 'mainLogTracker'.